site/en/docs/privacy-sandbox/fledge/index.md

```diff
@@ -475,3 +480,8 @@ We've written an  [API developer guide](/docs/privacy-sandbox/fledge-api) and bu
    Group](https://www.w3.org/community/web-adv/participants).
 -  **Developer support**: Ask questions and join discussions on the
    [Privacy Sandbox Developer Support repo](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support).
+-  **Current implementation**: For questions about the implementation currently available to test in
+   Chrome: [file a Chromium bug](https://bugs.chromium.org/p/chromium/issues/list?q=fledge).
+-  **Best practices**: For implementation, integration, and general best practice questions:
+   [create an issue](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support) 
+   on the Privacy Sandbox developer support repo.
```

Comment:
```suggestion
-  **Developer support**: Ask questions about implementation, and best practices, or join discussions on the
   [Privacy Sandbox Developer Support repo](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support).
-  **Current implementation**: For questions about the implementation currently available to test in
   Chrome: [file a Chromium bug](https://bugs.chromium.org/p/chromium/issues/list?q=fledge).
```

Reply:
I'd rather not link out to the same place twice as though it's two different places, so this groups those bullet points together.